client/sites/plan/components/current-plan/style.scss

```diff
@@ -0,0 +1,16 @@
+@import "@automattic/components/src/styles/typography";
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/variables";
+
+.current-plan {
```

Comment:
Should we add prefix here since .current-plan seems to exist already in https://github.com/Automattic/wp-calypso/blob/c5b41ef0653bb007b609fa3864f4d0c6a698cc32/client/my-sites/plans/current-plan/style.scss#L1?

Reply:
It belongs to the "My Plan" tab in the existing Plans page; if we're not loading that page shouldn't we be fine 🤔

Reply:
Looks like the CSS from that page is bundled, so it does indeed conflict 🤦 thanks, renaming...